packages/shared/src/governance.ts

```diff
@@ -879,17 +873,19 @@ export function governanceE2ETests<
   TCustom extends Record<string, unknown> | undefined,
   TInitStorages extends Record<string, Record<string, any>> | undefined,
 >(chain: Chain<TCustom, TInitStorages>, testConfig: { testSuiteName: string; addressEncoding: number }) {
-  describe(testConfig.testSuiteName, () => {
+  describe(testConfig.testSuiteName, async () => {
+    const [client] = await setupNetworks(chain)
```

Comment:
this means tests will share a same network, which maybe fine depends on the test content
if can sometimes caused unexpected result for example when run a single test vs run all the tests
what I sometimes do is record the current head, have a `beforeEach` step that setHead to previous head, effectively make the test environment clean